test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -304,6 +304,31 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(processesFound).To(BeEmpty(), "The node %s has the following processes on the root cgroup: %v", node.Name, processesFound)
 			}
 		})
+
+		It("Check for agetty process system.slice cgroup", func() {
```

Comment:
Lets phrase this with should  .  e.g. [It] should have agetty system service under the cpuset cgroup controller